src/Components/v2/ImageLink.tsx

```diff
@@ -0,0 +1,117 @@
+import { color, Flex, Serif, SerifSize } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Media } from "Utils/Responsive"
+
+export const ImageLink: React.SFC<ImageLinkProps> = ({
+  href,
+  imageUrl,
+  children,
+  fontSize,
+  ...rest
+}) => (
```

Comment:
Implicit returns tend to always get unwound in the future, as additions (variables, debugger statements) get added.

Reply:
👍 I probably did this in a rush and didn't notice it on commit. I'll fix.